django_celery_results/settings.py

```diff
@@ -1,16 +1,19 @@
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 
 
 def get_callback_function(settings_name, default=None):
     """Return the callback function for the given settings name."""
     
     callback = getattr(settings, settings_name, None)
-    if callback is None:
+    if not callback:
         return default
 
-    if callable(callback):
-        return callback
+    if not callable(callback):
+        raise ImproperlyConfigured(f"{settings_name} must be callable.")
+
+    return callback
 
 extend_task_props_callback = get_callback_function(
-    "CELERY_RESULTS_EXTEND_TASK_PROPS_CALLBACK"
+    "CELERY_RESULTS_EXTEND_TASK_PROPS_CALLBACK", dict
 )
```

Comment:
```suggestion
)

```

Reply:
@auvipy This is already done. I couldn't apply your suggestion because the rebase but the change is already there.